README.md

```diff
@@ -744,3 +745,64 @@ paths:
 The definitions section can be used by reference objects in order re-use specification code. If elements of the spec are copied and pasted more than once, they can be set here and reused.
 
 ----
+
+#### Define a Token to Access Dispatcher
+We allow services, which are published on Exosite Marketplace, to access [dispatcher APIs](https://pegasus-dispatcher.hosted.exosite.io).
+
+##### Step 1: define `x-exosite-token`
+In `schema`, add a attribute called `x-exosite-token`, the value can be any string,
+it will be used as the token when you try to call dispatcher.
+
+An example (refer to this [minimalservice.yaml](./examples/minimalservice.yaml)):
+```yaml
+# ...
+################################################################################
+#                            API Information                                   #
+################################################################################
+# ...
+x-exosite-token: "myprivatetoken"
+# ...
+```
+
+So, we defined a service called `minimalservice` and the token is `myprivatetoken`.
+
+##### Step 2: call dispatcher
+
+Add a header `authorization: minimalservice myprivatetoken` for all the following endpoints.
+
+- 2.1 call Event Trigger
+
+  base_url = https://pegasus-dispatcher.hosted.exosite.io
```

Comment:
```suggestion
  base_url = https://murano-service-api.hosted.exosite.io
```
@csrl  @johnlinvc  any suggestion on the public api domain?

Reply:
the current domain name is defined like this: [murano_ec2/development](https://github.com/exosite/murano_ec2/commit/e65470459c3d4c8749c401010c7381e4b0500373#diff-7741f37c8b6173cd1b798b99bd0cb48fR211),  should we change it to `serviceapis`? @johnlinvc  need your suggestions!!!

Reply:
Actually it would be better to be:
> murano-service-api.exosite.io